src/components/modal/readme.md

```diff
@@ -74,10 +74,12 @@ modal.beforeClose = beforeClose;
 
 ## Events
 
-| Event               | Description                                            | Type               |
-| ------------------- | ------------------------------------------------------ | ------------------ |
-| `calciteModalClose` | Emits when the component finishes the close animation. | `CustomEvent<any>` |
-| `calciteModalOpen`  | Emits when the component finishes the open animation.  | `CustomEvent<any>` |
+| Event                     | Description | Type                |
+| ------------------------- | ----------- | ------------------- |
+| `calciteModalBeforeClose` |             | `CustomEvent<void>` |
```

Comment:
@Elijbet can you investigate why the description doc is being removed from these events?

Reply:
Looks like the JSDoc comments were replaced by regular block ones: https://github.com/Esri/calcite-components/commit/4b3db60bb003b3c0e2b478bb1e10f65d9480469e#diff-1750b7bbce65a82bbddc6f7213b8f1cd41eec744c2d2ad386557f0d4d735e51bR291

Reply:
Yep, putting it on the to-do list.